docs/reference/tools-reference/azure_ai_language_tool.md

```diff
@@ -0,0 +1,148 @@
+# Azure AI Language
```

Comment:
Please kindly move the tool docs to be under https://github.com/microsoft/promptflow/tree/main/docs/integrations/tools

And fill in your tool contact info in the table.
https://github.com/microsoft/promptflow/blob/main/docs/integrations/tools/index.md